scripted-tests/run/execution-context/build.sbt

```diff
@@ -48,7 +48,8 @@ testQueueExecutionContext2 := {
 
 lazy val testEventLoop = taskKey[Unit]("...")
 testEventLoop := Def.taskDyn {
-  if (isWindows) Def.task { println("EvenLoop test skipped") }
+  // libuv is preintstalled only on MacOS GithubRunners
+  if (!isMac) Def.task { println("EvenLoop test skipped") }
```

Comment:
I suggest to disable this tests when multithreading is enabled. LibUV is not thread-safe and working with it from multithreading environment requires some additional work to garantue that you haven't modified something when it inside the loop from another thread.

Reply:
Multithreading is already disable for the tests in this scripted test. It's not shown here, but it's disabled at line 13

Reply:
I've missed that than.